packages/react/src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -18197,7 +18201,9 @@ Map {
       "wrapperClassName": null,
     },
     "propTypes": Object {
-      "addToList": [Function],
```

Comment:
Do you know why this one changed?

Reply:
it was marked as experimental feature and was wrapped by a warning function